README.md

```diff
@@ -23,4 +23,13 @@ Community contributed layouts.
 
 Community contributed widgets.
 
+### Animated Gif
+
+A widget that will run animated gifs.
+
+```go
+gif, err := NewAnimatedGif(storage.NewFileURI("./testdata/minions.gif"))
```

Comment:
What license is this / do we have permission from Universal Pictures to use this (or whoever owns the rights)? Might be better to use the rotating Earth from https://en.wikipedia.org/wiki/Gif which is under Creative Commons

Reply:
I downloaded it from a royalty free site. rotating earth is probably better, thanks.

Reply:
👍

This line needs to be updated;
```suggestion
gif, err := NewAnimatedGif(storage.NewFileURI("./testdata/gif/earth.gif"))
```

Reply:
Thanks, looks like the search missed that occurrence!